XLS-0065d-single-asset-vault/README.md

```diff
@@ -164,11 +165,19 @@ The `Vault` object costs one reserve fee per object created:
 
 Shares represent the portion of the Vault assets a depositor owns. Vault Owners set the currency code of the share and whether the token is transferable during the vault's creation. These two values are immutable. The share is represented by a [Multi-Purpose Token](https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-0033-multi-purpose-tokens). The MPT is issued by the vault's pseudo-account.
 
-##### 2.1.6.1 `MPTokenIssuance`
+##### 2.1.6.1 `AssetScale`
+
+Because vault shares are represented by MPTs, which must be whole numbers, any fractional part of a deposit gets truncated. For example, a deposit of 20.3 assets would be incorrectly credited as only 20 shares.
```

Comment:
We do not need this part "For example, a deposit of 20.3 assets would be incorrectly credited as only 20 shares." because it does not apply after this PR is merged.